#4 Adding Index Structure and Getting Started section (#6)
Inbibo/shift_docs: docs/intro/authoring.md

# Workflow Authoring

By opening Shift you will be prompted to an empty board.  

## Creating Your First Node

> To create a new operator, you can either press the `Tab Key` or `Double Left-Click` with your mouse.  

Reviewer: ```suggestion
To create a new [operator](intro/terminology.md/#operator), you can either press the `Tab Key` or `Double Left-Click` with your mouse.  
```

![Create Node Dialog](../images/node_creation_dialog.png "Create Node Dialog")  

Then you can choose the type of operator you want to create and double-click on its name to create it.

## Creating Connections

> If you want a certain operator to pass information to another one, you can connect their plugs. To do that, you just need to `Left-Click and Drag` from one plug to the other plug you want to connect to. This will propagate whatever data is produced by the first operator and pass it to the connected plug.  

Reviewer: ```suggestion
If you want a certain operator to pass information to another one, you can connect their [plugs](intro/terminology.md/#plug). To do that, you just need to `Left-Click and Drag` from one plug to the other plug you want to connect to. This will propagate whatever data is produced by the first operator and pass it to the connected plug.  
```

Author: Done!

![Connected Nodes](../images/connected_nodes.png "Connected Nodes")  

This will have created the simplest example of a Shift workflow.

## Creating New Plugs

Certain nodes in Shift allow the creation of new plugs. To do that you can `Right-Click` on the node and select the "*Create New Plug*" option if available. It will prompt you to a new dialog asking to fill in the information needed to create the plug.

![New Plug](../images/new_plug.gif)

You can also prompt the creation of a new plug while creating a connection. This will automatically fill in the name and type, matching the plug that the connection is coming from.
To do that simply release your `Left-Click` mouse button over the node when creating the connection.

![New Plug](../images/new_plug_drag_connection.gif)

## Removing a Plug

> To remove a plug you can use the `Right-Click` context menu of the plug item. Removing a plug will destroy all the existing connections to that plug.

Reviewer: ```suggestion
To remove a plug you can `Right-Click` on the plug and select `Renove Plug`. Removing a plug will destroy all the existing connections to that plug.
```

Author: Apart from the typo I would change it like this. The "` `" formatting is reserved for keys or commands. *"Remove Plug"* will display it in cursive. What do you think?
```suggestion
To remove a plug you can `Right-Click` on the plug and select *"Remove Plug"*. Removing a plug will destroy all the existing connections to that plug.
```

Author: Applied my suggested change.
